client/src/js/partialuserforms/partialsubmission.js

```diff
@@ -33,7 +37,10 @@ const submitPartial = () => {
     const fieldName = element.getAttribute('name');
     const value = getElementValue(element, fieldName);
     if (!data.has(fieldName)) {
-      if (typeof value === 'object') {
+      if (typeof value === 'object' && element.getAttribute('type') === 'file') {
+        data.append(fieldName, value);
+      }
```

Comment:
Can we move `else if` in the same line as the closing bracket

Reply:
Funny the linter didn't pick up on that!